packages/in_app_purchase/in_app_purchase_ios/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -64,6 +65,18 @@ class SKPaymentQueueWrapper {
     channel.setMethodCallHandler(_handleObserverCallbacks);
   }
 
+  /// Call this when the first listener is subscribed to the
+  /// [InAppPurchaseIosPlatform.purchaseStream].
```

Comment:
I think it would be better to first describe the purpose / intent of the method and after that how to use it. This will give users an idea on why they should use it.

```suggestion
  /// Instructs the iOS implementation to register a transaction observer and
  /// start listening to it.
  /// 
  /// Call this method when the first listener is subscribed to the
  /// [InAppPurchaseIosPlatform.purchaseStream].
```